src/commands/lambda/functions/instrument.ts

```diff
@@ -135,16 +146,21 @@ export const calculateUpdateRequest = async (
   let needsUpdate = false
 
   // Update Handler
-  const expectedHandler = HANDLER_LOCATION[runtime]
-  if (config.Handler !== expectedHandler) {
-    needsUpdate = true
-    updateRequest.Handler = HANDLER_LOCATION[runtime]
+  if (runtime !== DOTNET_RUNTIME) {
+    const expectedHandler = HANDLER_LOCATION[runtime]
+    if (config.Handler !== expectedHandler) {
+      needsUpdate = true
+      updateRequest.Handler = HANDLER_LOCATION[runtime]
+    }
   }
 
   // Update Env Vars
-  if (oldEnvVars[LAMBDA_HANDLER_ENV_VAR] === undefined) {
-    needsUpdate = true
-    changedEnvVars[LAMBDA_HANDLER_ENV_VAR] = config.Handler ?? ''
+  // Don't add dotnet
```

Comment:
What do you mean by this comment `Don't add dotnet`?

Reply:
We don't need to add the handler env var for dotnet. I added more detail to the comment.